src/common/events/event_builders.cpp

```diff
@@ -240,6 +239,33 @@ void mev::add_touch(
     tev->set_action(current_index, action);
 }
 
+mir::EventUPtr mev::make_pointer_event(
+    MirInputDeviceId device_id,
+    std::chrono::nanoseconds timestamp,
+    std::vector<uint8_t> const& cookie,
+    MirInputEventModifiers mods,
+    MirPointerAction action,
+    MirPointerButtons buttons,
+    std::optional<geom::PointF> position,
+    geom::DisplacementF motion,
+    MirPointerAxisSource axis_source,
+    events::ScrollAxisV1H h_scroll,
+    events::ScrollAxisV1V v_scroll)
```

Comment:
I think it's important for the next person who's reading this to document that this is *deliberately* using the `V1` types while the header declares these parameters with the unversioned alias so that we get a compile error should someone bump the unversioned alias without remembering to do the rest of the work to maintain ABI.